packages/blocks/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -0,0 +1,130 @@
+import { PlusIcon, SearchIcon } from '@mergestat/icons';
+import React, { useState } from 'react';
+import { Tag } from '../Filter/Tag';
+
+type MultiSelectProps = {
+  setStateToProps: {
+    title: string;
+    checked: boolean;
+  }[];
+  getState?: ([]) => void;
+};
+
+export const MultiSelect: React.FC<
+  MultiSelectProps &
+    React.DetailedHTMLProps<
+      React.DetailsHTMLAttributes<HTMLElement>,
+      HTMLElement
+    >
+> = ({ setStateToProps, getState }) => {
+  const [state, setState] = useState(setStateToProps);
+  const [value, setValue] = useState('');
+  const [isActive, setIsActive] = useState(false);
+
+  window.addEventListener('click', (e) => {
+    const target = e.target as Element
+    if (target.localName !== 'input' && target.localName !== 'label') {
+      setIsActive(false);
+    }
+  });
+
+  window.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter' && value.length > 0) {
+      setState([...state, { title: value, checked: true }]);
+
+      if (getState) {
+        getState([...state]);
+      }
+      setValue('');
+    }
+  });
+
+  return (
+    <div className="t-container">
+      <div className="t-tags-container">
+        {state.map((item, index) => {
+          if (item.checked) {
+            return (
+              <Tag
+                skin="gray"
+                key={`key_${index}`}
+                onClick={() => {
+                  state[index].checked = !state[index].checked;
+                  setState([...state]);
+                  if (getState) {
+                    getState([...state]);
+                  }
+                }}
+              >
+                {item.title}
+              </Tag>
+            );
+          }
+        })}
+      </div>
+      <div className={`t-search-container ${isActive && 't-shadow'}`}>
+        <label
+          className={`t-inptut-container  ${isActive && 'active'}`}
```

Comment:
this looks like a typo - `t-inptut-container` should be `t-input-container` wherever it's defined